src/hugr/serialize.rs

```diff
@@ -57,6 +58,21 @@ struct SerHugrV0 {
     metadata: Vec<serde_json::Value>,
 }
 
+/// Version 1 of the HUGR serialization format.
+#[derive(Serialize, Deserialize, PartialEq, Debug)]
+struct SerHugrV1 {
```

Comment:
I suppose we don't need `SerHugrV0` any more and can delete the struct definition?